client/components/Modals/HouseRulesModal.jsx

```diff
@@ -15,6 +87,22 @@ const HouseRulesModal = ({ toggleModal }) => {
 
   const handleClickInside = () => setClickedOutside(false);
 
+  const rules = houseRules.map((houseRule, i) => {
+    let wordArray = houseRule.safetyDescription.split(' ');
+    let lastWord = wordArray[wordArray.length - 1].toLowerCase();
+    return (
+      <span key={houseRule._id}>
+        <Paragaraph>
+          <img
+            src={`${Url}${lastWord}.png`}
+            style={{ maxHeight: '15px' }}
+          ></img>{' '}
+          {houseRule.safetyDescription}
+        </Paragaraph>
+      </span>
+    );
+  });
+
   useEffect(() => {
     // Check for outside clicks, and prevent scrolling when the modal is open
     document.addEventListener('mousedown', handleClickOutside);
```

Comment:
Sorry, just saw this. I have yet to play around with hooks!

Reply:
I love them. Def want to play around with them more. Maybe during solo week???